testdata/kubecfg_test.jsonnet

```diff
@@ -78,6 +78,25 @@ local result =
 
   std.assertEqual(std.clamp(42, 0, 10), 10) &&
 
+  local testRenderOverlay = {
+    a: {
+      b: {
+        c: 1,
+        d: 10,
+      },
+    },
+  };
+  local expectedOverlayObj = {
+    a: {
+      b: {
+        c: 2,
+        d: 10,
+      },
+    },
+  };
+  std.assertEqual(kubecfg.renderOverlay(testRenderOverlay, import 'overlay.json'), expectedOverlayObj) &&
```

Comment:
This doesn't actually work at the moment, I'm trying to figure out why :thinking: 

```
kubecfg show kubecfg_test.jsonnet
ERROR error reading kubecfg_test.jsonnet: RUNTIME ERROR: Field does not exist: renderOverlay
```

Reply:
Nevermind, I'm dumb :facepalm: 

I'm using `kubecfg` compiled myself which doesn't have the updated `lib/kubecfg.libsonnet`